client/post-editor/editor-feedback-invitation/index.jsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { identity } from 'lodash';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+
+function EditorFeedbackInvitation( { translate, onTrigger } ) {
+	return (
+		<div className="editor-feedback-invitation">
+			<p className="editor-feedback-invitation__invitation">
+				{ translate(
+					'{{highlight}}Need another set of eyes?{{/highlight}} Ask a friend to review your post.',
+					{
+						components: {
+							highlight: <span className="editor-feedback-invitation__highlight" />
+						}
+					}
+				) }
+			</p>
+			<Button className="editor-feedback-invitation__button" onClick={ onTrigger }>
+				{ translate( 'Send to a Friend' ) }
+			</Button>
+		</div>
+	);
+}
```

Comment:
Well `function` still has a few advantages, namely that it's statically available whereas `const` expressions need to be defined before they are used. I wonder too at the way the engine handles them differently too but my own style leads me to prefer the arrow functions and "wait and see" if there are bigger issues.

It's true that the dev tools have improved to support the patterns in use which is a good thing!